l10n_ch_qr_bill/models/res_bank.py

```diff
@@ -24,6 +24,18 @@ class ResPartnerBank(models.Model):
         )
     )
 
+    @api.multi
+    @api.constrains('l10n_ch_qr_iban')
```

Comment:
You constrain depending on ```l10n_ch_qr_iban``` but you are working in the code only with ```l10n_ch_qrr```. Does ```l10n_ch_qr_iban``` depend on the value set on ```l10n_ch_qrr```?

Reply:
Thanks carlos, this was a mistake. Changing. it should be `l10n_ch_qr_iban` used in the method.